docs/version-switcher.js

```diff
@@ -0,0 +1,48 @@
+(async function () {
```

Comment:
Double check that this file is identical to:
https://github.com/E3SM-Project/polaris/blob/main/docs/shared/version-switcher.js